code/modules/economy/account.dm

```diff
@@ -6,6 +8,8 @@
 	var/add_to_accounts = TRUE
 	var/account_id
 	var/welfare = FALSE
+	var/being_dumped = FALSE //pink levels are rising
+	var/canWithdraw = FALSE
```

Comment:
This isn't a boo :(

Reply:
what did cobby mean by this

Reply:
canWithdraw is not a `TRUE` or `FALSE` so just put 0 and/or rename the var.

Reply:
I think he means it's not a boolean

Reply:
wtf i made that var who set its value to that
smh

Reply:
ty yall. done